x-pack/legacy/plugins/spaces/public/views/management/components/confirm_delete_modal.tsx

```diff
@@ -174,18 +174,18 @@ class ConfirmDeleteModalUI extends Component<Props, State> {
   private onSpaceNameChange = (e: ChangeEvent<HTMLInputElement>) => {
     if (typeof this.state.error === 'boolean') {
       this.setState({
-        confirmSpaceName: e.target.value,
+        confirmDelete: e.target.value,
         error: e.target.value !== this.props.space.name,
       });
     } else {
       this.setState({
-        confirmSpaceName: e.target.value,
+        confirmDelete: e.target.value,
       });
     }
   };
 
   private onConfirm = async () => {
-    if (this.state.confirmSpaceName === this.props.space.name) {
+    if (this.state.confirmDelete === 'DELETE') {
```

Comment:
We'll want to make this friendlier for folks using languages other than English. Rather than checking for `DELETE` explicitly, I think it'd make sense to check for an internationalized version of the text instead. Something like this (untested):

```ts
const deleteText = intl.formatMessage({
    id: 'xpack.spaces.management.confirmDeleteModal.deleteText',
    defaultMessage: 'DELETE',
});

if (this.state.confirmDelete === deleteText) {
 // ...
}
```


Similarly above, we'll want to tweak the message to accept an internationalized string:
```ts
intl.formatMessage({
    id: 'xpack.spaces.management.confirmDeleteModal.confirmSpaceNameFormRowLabel',
    defaultMessage: 'Please type {deleteText} to delete your space.'
}, {
    deleteText
})'
```

Reply:
Quick question, is this capitalization aware as well? Meaning will `delete` work as well?

Reply:
Same comments as above:

To delete the space, enter {deleteText}.